.github/workflows/test_e2eshark.yml

```diff
@@ -57,14 +59,18 @@ jobs:
 
       - name: Run Onnx Mode
         run: |
+          echo $AZ_PRIVATE_CONNECTION
+          echo $HF_TOKEN
+          echo $E2E_VENV_DIR
```

Comment:
Remove these lines. https://docs.github.com/en/actions/security-guides/using-secrets-in-github-actions#accessing-your-secrets

> Warning: If a secret was used in the job, GitHub automatically redacts secrets printed to the log. You should avoid printing secrets to the log intentionally.

Reply:
Thanks, forgot that was even there :)